.azure-pipelines.yaml

```diff
@@ -0,0 +1,69 @@
+pr: none
+
+jobs:
+- job:
+  displayName: Linux
+  pool:
+    vmImage: "ubuntu-16.04"
+
+  steps:
+  - bash: echo "##vso[task.prependpath]$CONDA/bin"
+    displayName: Add conda to PATH.
+
+  - bash: conda update conda --yes --quiet
+    displayName: Update conda.
+
+  - bash: conda env create
```

Comment:
Make explict by adding ` -f environment.yml` ?

Reply:
Sure, I will also set the name.